sentry_sdk/integrations/httpx.py

```diff
@@ -52,13 +53,18 @@ def send(self, request, **kwargs):
             span.set_data("http.query", parsed_url.query)
             span.set_data("http.fragment", parsed_url.fragment)
 
-            for key, value in hub.iter_trace_propagation_headers():
-                logger.debug(
-                    "[Tracing] Adding `{key}` header {value} to outgoing request to {url}.".format(
-                        key=key, value=value, url=request.url
+            if should_propagate_trace(
+                str(request.url),
```

Comment:
What is the difference between `parsed_url.url` and this?

Reply:
`parsed_url` is a named tuple `(url, query_string, fragment)`
And the the `parsed_url.url` is just the base url without query_string and fragment